site/content/data-platform/installation.md

```diff
@@ -0,0 +1,448 @@
+the `PATH` environment variable to make it available as a command in the system.
+
+The Platform CLI tool simplifies the further setup and later management of
+the Platform's Kubernetes services.
+
+### Step 8: Install the Data Platform package
+
+Install the package using the package configuration you received from the
+Arango team (`platform.yaml`).
+
+The package installation creates and enables various services, including
+the unified web interface of the Data Platform.
+
+Substitute `<license-client-id>` and `<license-client-secret>`
+with the actual license credentials and `./platform.yaml` with the path to the
+package configuration file.
+
+```sh
+arangodb_operator_platform --context arangodb package install \
+  --license.client.id "<license-client-id>" \
+  --license.client.secret "<license-client-secret>" \
+  --platform.name deployment-example \
+  ./platform.yaml
+```
+
+It can take a while to run this command because it downloads the Platform Suite,
+and in case of the AI Data Platform, also the AI Suite.
+
+### Step 9: Set up object storage
+
+Features like GraphML require an additional storage system to save model
+training data, for instance.
+
+The following example shows how to set up a local MinIO and integrate it with
+the Arango Data Platform, but you can also use a remote object storage like S3.
+For the supported storage systems, see the
+[`kube-arangodb` documentation](https://arangodb.github.io/kube-arangodb/docs/platform/storage.html).
+
+Create a Kubernetes namespace for MinIO, then create a secret in this namespace
+with the username and password to use for the MinIO root user (replace `minioadmin`
+and `miniopassword` with the credentials you actually want to use). Create another
+secret with the same credentials but in the namespace of your `ArangoDeployment`,
+which is `arangodb` in this example:
+
+```sh
+kubectl create namespace minio
+
+kubectl create secret generic minio-root \
+  --namespace minio \
+  --from-literal=MINIO_ROOT_USER=minioadmin \
+  --from-literal=MINIO_ROOT_PASSWORD=miniopassword
+
+kubectl create secret generic minio-credentials \
+  --namespace arangodb \
+  --from-literal=accessKey=minioadmin \
+  --from-literal=secretKey=miniopassword
+```
+
+Create a file to configure MinIO service and call it e.g. `minio.yaml`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: minio
+  namespace: minio
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: minio
+  template:
+    metadata:
+      labels:
+        app: minio
+    spec:
+      containers:
+      - name: minio
+        image: minio/minio:latest
+        args:
+          - server
+          - /data
+        envFrom:
+          - secretRef:
+              name: minio-root
+        ports:
+          - containerPort: 9000
+        volumeMounts:
+          - name: data
+            mountPath: /data
+      volumes:
+        - name: data
+          emptyDir: {}
```

Comment:
When I setup the minio service locally, I wasn't able to successfully upload any file to minio using `emptyDir`. Instead, I had to create a PersitentVolumeClaim in addition, and configured the `volumes` section as follows:

```
      volumes:
        - name: data
          persistentVolumeClaim:
            claimName: minio-data-pvc
 ```
 
 minio-pvc.yaml
 ```
 apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: minio-data-pvc
  namespace: minio
spec:
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 5Gi
```